README.md

```diff
@@ -23,6 +23,94 @@ or solve build problems, we would be happy to accept your contributions.
 
 ## How to Run Basekernel
 
+### Installing a cross-compiler
+
+To avoid potential system-dependent behavior changes, you will probably
+want to use a cross-compiler. A cross-compiler is a compiler intended to
+run on one system, but compile code for a different one. In this case,
+your computer is the former system ("host"), and basekernel is the
+latter system ("target").
+
+The Makefile as-is coerces the compiler to build for an x86 target
+anyway, but it is preferable to use a version of GCC actually 
+pre-configured for x86 at build time.
+
+We hope to eventually move from this coersion and just expect the user
+to have a cross-compiler, so any feedback about your experience with
+these instructions is greatly appreciated.
+
+#### Building from sources
+
+To build from sources (installs in $HOME directory):
+
+```
+# get and extract sources
+curl -O https://ftp.gnu.org/gnu/binutils/binutils-2.29.1.tar.gz
+curl -O https://ftp.gnu.org/gnu/gcc/gcc-7.2.0/gcc-7.2.0.tar.gz
+tar -zxf gcc-7.2.0.tar.gz
+tar -zxf binutils-2.29.1.tar.gz
+
+# build and install binutils
+cd binutils-2.29.1
+./configure --prefix=$HOME/cross --target=i386-elf \
```

Comment:
You should wrap the prefix path in double quotes in case you have characters that need escaping (e.g. spaces).

Change it to `--prefix="$HOME/cross"`.